source/__tests__/integration/export-profile.test.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path')
+
+describe('Sinopia Profile Editor exports a loaded Profile', () => {
+
+  beforeAll(async () => {
+    await page.goto('http://127.0.0.1:8000/#/profile/create')
+  })
+
+  it('clicks Verbose Export and displays an error with missing profile form data', async () => {
```

Comment:
sharing a relevant story:  DLSS was fortunate to hire Sandi Metz to teach us a week long class in coding practices.  At one point, one of us put some comments in the code we were refactoring.  Sandi asked said person to "read the code out loud" ... and it said just what the comment said ... showing the comment was actually useless noise.

The "clicks Verbose Export" might be in the same category - it's pretty clear on line 10.

A better description might be "error displayed when exporting empty profile" or some such.

This is not a blocker -- just something to keep in mind when describing tests.